feature/onboarding/src/main/kotlin/net/thunderbird/feature/onboarding/OnboardingContent.kt

```diff
@@ -0,0 +1,165 @@
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import app.k9mail.core.ui.compose.common.DevicePreviews
+import app.k9mail.core.ui.compose.designsystem.atom.Background
+import app.k9mail.core.ui.compose.designsystem.atom.button.Button
+import app.k9mail.core.ui.compose.designsystem.atom.button.ButtonText
+import app.k9mail.core.ui.compose.designsystem.atom.text.TextBody1
+import app.k9mail.core.ui.compose.designsystem.atom.text.TextHeadline2
+import app.k9mail.core.ui.compose.designsystem.template.LazyColumnWithFooter
+import app.k9mail.core.ui.compose.designsystem.template.ResponsiveContent
+import app.k9mail.core.ui.compose.theme.K9Theme
+import app.k9mail.core.ui.compose.theme.MainTheme
+import app.k9mail.core.ui.compose.theme.ThunderbirdTheme
+
+@Composable
+internal fun OnboardingContent(
+    onStartClick: () -> Unit,
+    onImportClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    ResponsiveContent {
+        Background(
+            modifier = modifier,
+        ) {
+            LazyColumnWithFooter(
+                modifier = Modifier
+                    .fillMaxHeight()
+                    .padding(MainTheme.spacings.double),
+                footer = {
+                    WelcomeFooter(
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .padding(top = MainTheme.spacings.triple),
+                        onStartClick = onStartClick,
+                        onImportClick = onImportClick,
+                    )
+                },
+                verticalArrangement = Arrangement.spacedBy(MainTheme.sizes.large, Alignment.CenterVertically),
+            ) {
+                item {
+                    WelcomeLogo(
+                        modifier = Modifier.fillMaxWidth(),
+                    )
+                }
+                item {
+                    WelcomeTitle(
+                        modifier = Modifier.fillMaxWidth(),
+                    )
+                }
+                item {
+                    WelcomeMessage(
+                        modifier = Modifier.fillMaxWidth(),
+                    )
+                }
+            }
+        }
+    }
+}
+
+@Composable
+private fun WelcomeLogo(
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = Modifier.then(modifier),
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+        Box(
+            modifier = Modifier
+                .size(MainTheme.sizes.huge)
+                .clip(shape = RoundedCornerShape(percent = 50))
+                .background(color = MainTheme.colors.surface),
+        ) {
+            Image(
+                painter = painterResource(id = MainTheme.images.logo),
+                contentDescription = stringResource(id = R.string.onboarding_welcome_logo_content_description),
```

Comment:
This feels like a purely decorative image, i.e. we don't want a content description for this.